source/manual/readmes.html.md

```diff
@@ -7,105 +7,73 @@ type: learn
 parent: "/manual.html"
 ---
 
-This is a guide to writing and maintaining README documents for GOV.UK's public repositories.
+This is a guide to writing and maintaining README documents for GOV.UK's public repositories. It's based on [guidance we have in the GDS Way for READMEs](https://gds-way.cloudapps.digital/manuals/readme-guidance.html#writing-readmes).
 
-## Who are READMEs for (aka what's the user need)?
+READMEs are for a technical audience. This could be a new starter or an existing developer. Using the following template will help to ensure each README is consistent, correct, short and actionable.
 
-- New starters to GDS
-- People joining a new team
-- People interested in our work
-- Developers on other teams
+Unless it helps someone to get started, any other content should be a separate file in the `docs/` directory for a repository. This also makes the content easier to discover in external search.
 
 ## Template for new READMEs
 
 ```markdown
 # App name
 
-One paragraph description and purpose.
+One or more paragraphs describing the app:
 
-## Screenshots (if there's a client-facing aspect of it)
+- What it's used for.
+- Any unusual dependencies it has.
 
-## Live examples (if available)
+## Live examples
 
-- [gov.uk/thing](https://www.gov.uk/thing)
+(Only applies to frontend apps.)
+
+A list of links to example pages rendered by the app.
 
 ## Nomenclature
 
-- **Word**: definition of word, and how it's used in the code
+A list of definitions for unusual terms in the code.
 
 ## Technical documentation
 
-Write a single paragraph including a general technical overview of the app.
-Example:
-
-This is a Ruby on Rails application that maps RESTful URLs onto a persistence
-layer. It's only presented as an internal API and doesn't face public users.
-
-### Dependencies
-
-- [alphagov/other-repo]() - provides some downstream service
-- [redis]() - provides a backing service for work queues
-
-### Running the application
+What goes here depends on the type of app.
 
-`./one-line-command.sh`
-
-Documentation for where the app will appear (default port, vhost, URL etc).
+See the links below for example content to put here.
 
 ### Running the test suite
 
-`./one-line-command.sh`
-
-Include any other edge cases, e.g parallel test runner in Whitehall
+Give one command to run all the tests, linting, etc.
```

Comment:
Would also include information about error messages and suggested fixes

Reply:
I agree that can be helpful, but only if the errors are frequent. We don't really have an example of that though, because if an error is really that bad then we would just go and fix it. For less frequent errors, we could suggest a separate "troubleshooting" guide in the "further docs" section. I'm keen not to make people read a bunch of stuff that's only relevant once in a blue moon. A troubleshooting guide has the advantage of single, clear purpose.

How does that sound?

Reply:
I would recommend against a troubleshooting section. It's content best practice to have issues in the section of docs that those issues relate to. For example, https://docs.notifications.service.gov.uk/java.html#error-codes.

What I would recommend is to have a clear route for users to support on the less frequent errors, as per my next comment. That way, when a user does have an error, they can go down the support route and get it fixed.

Reply:
Ah, I think I see what you mean. GOV.UK apps don't have direct users - everything is provided as a service to publishers and to the public. So the only people we need to "support" are the same devs who wrote the code.

Content about error messages or general usage instructions would definitely be appropriate for a library, but this template is specific to apps, so I don't think we need to worry about it in here.

This is perhaps one example of where having a GOV.UK app-specific template can be helpful to avoid ambiguity.

Does clarifying the users help at all?

Reply:
ok, that does help. no need to have information about errors and fixes then